modal/_utils/rand_pb_testing.py

```diff
@@ -58,12 +58,10 @@ def _fill(msg, desc: Descriptor, rand: Random) -> None:
         else:
             if field.type == FieldDescriptor.TYPE_ENUM:
                 enum_values = [x.number for x in field.enum_type.values]
-
-                def generator(rand):
-                    return rand.choice(enum_values)
+                generator = lambda rand: rand.choice(enum_values)  # noqa: E731
```

Comment:
@freider I added a ruff ignore here to avoid having a `def`, which requires a function type signature and conflicts vs line 64.